plugins/device/store-utils.go

```diff
@@ -13,18 +13,28 @@ func GetDevices(peerID string) ([]deviceapi.Info, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(peerInfo.MetaData["devices"]) > 0 {
+	if len(peerInfo.Metadata["_devices"]) > 0 {
 		var deviceInfo []deviceapi.Info
-		if err := json.Unmarshal([]byte(peerInfo.MetaData["devices"]), &deviceInfo); err != nil {
+		if err := json.Unmarshal([]byte(peerInfo.Metadata["_devices"]), &deviceInfo); err != nil {
 			return nil, err
 		}
 		return deviceInfo, nil
 	}
 	return nil, nil
 }
 
-// AddDevices adds device to specific peer
-func AddDevices(devices []deviceapi.Info, peerID string) error {
+//CheckIfDeviceExist returns error if all devices already exist or returns list of devices to be added
```

Comment:
Incorrect comment: This function returns true if **any** of the devices exist.

Reply:
Also, do not export this function until you have an external consumer out of this package.